src/cgroup.rs

```diff
@@ -326,11 +344,7 @@ pub fn get_cgroups_relative_paths() -> Result<HashMap<String, String>> {
         for key in &keys {
             // this is a workaround, cgroup file are using `name=systemd`,
             // but if file system the name is `systemd`
-            if *key == "name=systemd" {
-                m.insert("systemd".to_string(), fl[2].to_string());
-            } else {
-                m.insert(key.to_string(), fl[2].to_string());
-            }
+            m.insert(key.to_string(), fl[2].to_string());
```

Comment:
Why  rm those  systemd case?

Reply:
The systemd hierarchy name should just exactly be `name=systemd` not `systemd`